docs/ViewModel.md

```diff
@@ -16,69 +16,39 @@ be created:
 
 ```js
 var MyTagViewModel = DefineMap.extend("MyTagViewModel",{
-    message: "string"
+	message: "string"
 });
 
 Component.extend({
-    tag: "my-tag",
-    ViewModel: MyTagViewModel,
-    view: stache("<h1>{{message}}</h1>")
+	tag: "my-tag",
+	ViewModel: MyTagViewModel,
+	view: stache("<h1>{{message}}</h1>")
 })
 ```
 
 Use [can-view-model] to read a component's view model instance.
 
-  @param {Object} properties The initial properties that are passed by the [can-stache-bindings data bindings].
+@param {Object} properties The initial properties that are passed by the [can-stache-bindings data bindings].
 
-  The view bindings on a tag control the properties and values used to instantiate the `ViewModel`. For example, calling `<my-tag>` as follows invokes `MyTagViewModel` as shown in the comments:
+The view bindings on a tag control the properties and values used to instantiate the `ViewModel`. For example, calling `<my-tag>` as follows invokes `MyTagViewModel` as shown in the comments:
 
-  ```
-  <my-tag/> <!-- new MyTagViewModel({}) -->
-
-  <my-tag
-      {message}="'Hi There'"/> <!-- new MyTagViewModel({message: "Hi There"}) -->
-
-  ```
-
-  @return {Object} A new instance of the corresponding constructor function. This instance is
-  added to the top of the [can-view-scope] the component's [can-component::view] is rendered with.
-
-@type {Object} A short hand for the prototype methods and properties used to extend the
-[can-util/js/types/types.DefaultMap default Map type] (typically [can-define/map/map]) and use
-that anonymous type as the view model.
-
-The following:
-
-```js
-Component.extend({
-    tag: "my-paginate",
-    ViewModel: {
-        offset: {value: 0},
-        limit: {value: 20}
-    }
-})
 ```
+<my-tag/> <!-- new MyTagViewModel({}) -->
 
-This is shorthand for:
+<my-tag
+	{message}="'Hi There'"/> <!-- new MyTagViewModel({message: "Hi There"}) -->
 
-```js
-var AnonymousViewModel = DefineMap.extend({
-    offset: {value: 0},
-    limit: {value: 20}
-})
-
-Component.extend({
-    tag: "my-paginate",
-    ViewModel: AnonymousViewModel
-})
 ```
 
+@return {Object} A new instance of the corresponding constructor function. This instance is
```

Comment:
I'm not sure there if `@return` should be here.  I guess I'd make sure that we make it clear that this function is called with `new`.  If that's the case, it's almost a given that the instance is going to be returned.

I'm not sure this needs to change though ...